packages/client3/components/design-system/input/text-area.tsx

```diff
@@ -65,13 +67,14 @@ export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
               ? "text-lg"
               : textSize === "xl"
               ? "text-2xl"
-              : null
+              : null,
+            outerClassName
           )}
         >
           <div
             className={clsx(
               "relative mt-1 w-full rounded",
-              focus ? "ring-2 ring-green-50" : "ring ring-dark-80",
```

Comment:
I'll defer to you, but I put ring so that it would not "jump" around on focus. I was noticing that the addition of the 2px border would have the box move by 2x2 px, where as tailwind's `ring` took care of that..

Reply:
Ahh, that makes sense. I removed it on the thought that the text field in design system didn't have ring around it